src/views/Dashboard.vue

```diff
@@ -248,15 +251,21 @@ export default {
       return this.isFirm && this.hasMissingInfoWarning
     },
 
-    /** Show Not In Good Standing alert only for firm with compliance warning. */
-    isNotInGoodStandingAlert (): boolean {
+    /** Show Not In Compliance alert only for firm with compliance warning. */
+    isNotInComplianceAlert (): boolean {
       return this.isFirm && this.hasComplianceWarning
     },
```

Comment:
This is the wrong logic or the wrong alert, so I fixed it (re: the new compliance alert component).